spk/sickchill/src/service-setup.sh

```diff
@@ -42,20 +42,28 @@ EOF
 service_postinst() {
     # Create a Python virtualenv
     ${VIRTUALENV} --system-site-packages ${SYNOPKG_PKGDEST}/env
-
+    
+    # attempt to get current pip updated during install procedure
+    ${PYTHON} -m pip install --upgrade pip
+    
     # Install the wheels
-    ${PIP} install --no-deps --no-index -U --force-reinstall -f ${SYNOPKG_PKGDEST}/share/wheelhouse ${SYNOPKG_PKGDEST}/share/wheelhouse/*.whl
-
+    wheelhouse=${SYNOPKG_PKGDEST}/share/wheelhouse
+    ${PIP} install --extra-index-url https://wheel-index.linuxserver.io/ubuntu/ --upgrade --force-reinstall --find-links ${wheelhouse} ${wheelhouse}/*.whl
```

Comment:
I'm curious what is the `--extra-index-url` for?

Reply:
This was an initial suggestion from the SickChill team as there were lots of build issues with cryptography and this site has builds cp38 of crypography not on Pypi.org. So just giving those poor old DSM a chance to look here. Not sufficiently knowledge on this, so probably unnecessary. But my assumption was that if we install and have this here then the DSM can see if there are valid newer versions of some packages for their particular hardware not provided on pypi e.g. SickChill looks for cryptography 3.4.7.

Reply:
Thanks for the explanation, yea the less the DSM has to compile the better :+1: